webapp/components/interop.js

```diff
@@ -716,7 +659,40 @@ class Interop2022 extends PolymerElement {
   }
 
   getBrowserScoreTotal(browserIndex) {
-    return this.getBrowserScoreForFeature(browserIndex, SUMMARY_FEATURE_NAME);
+    return this.totals[browserIndex];
+  }
+
+  getAllYears() {
+    const firstYear = 2021;
+    const currentYear = new Date().getFullYear();
+    const years = [];
+    for (let i = firstYear; i <= currentYear; i++) {
+      years.push(i.toString());
+    }
+    return years;
+  }
+
+  updateYearInfo(year) {
+    const yearInfo = PARAMS_BY_YEAR[year];
+    this.focusAreas = yearInfo.focus_areas;
+    this.summaryFeatureName = yearInfo.summary_feature_name;
+    this.tableSections = yearInfo.table_sections;
+    this.prose = yearInfo.prose;
+    this.issueURL = yearInfo.issue_url;
+    this.matrixURL = yearInfo.matrix_url;
+    this.features = Object.entries(this.focusAreas).map(([id, info]) => {
+      return Object.assign({ id }, info);
+    });
+  }
+
+  updateTotals(features) {
```

Comment:
1. Stable and Experimental data [are both obtained](https://github.com/web-platform-tests/wpt.fyi/blob/8e01960ba2a93ecff4b6167161aa1a43f20c4ff6/webapp/components/interop.js#L29-L31) at the time of page load, so we should be certain that we're seeing as up-to-date information as we need.

2. The page is redirected if navigating to a new year, which will pull the data for that specific year only (experimental and stable).

Reply:
> The page is redirected if navigating to a new year,

Aha that answers my questions. I thought switching tab reloads the data, instead of reloading the page